osf_pigeon/pigeon.py

```diff
@@ -36,17 +36,20 @@ async def dump_json_to_dir(from_url, to_dir, name, parse_json=None):
 
 
 def create_zip(temp_dir):
-    with zipfile.ZipFile(os.path.join(temp_dir, "bag.zip"), "w") as fp:
-        for root, dirs, files in os.walk(temp_dir):
-            for file in files:
-                file_path = os.path.join(root, file)
-                file_name = re.sub(f"^{temp_dir}", "", file_path)
-                fp.write(file_path, arcname=file_name)
+    files_to_zip = []
+    for root, dirs, files in os.walk(temp_dir):
+        for file in files:
+            file_path = os.path.join(root, file)
+            file_name = re.sub(f"^{temp_dir}", "", file_path)
```

Comment:
Correct that this is used as the "path" to the file in the archive? If so, can we come up with a better name?

This could also be computed on-the-fly when building up the zip instead of building and unpacking a list of tuples.

Reply:
> This could also be computed on-the-fly when building up the zip instead of building and unpacking a list of tuples. 

It's actually done in two steps so it doesn't zip add itself into the zip. Simply going to fix that by putting everything that's going to be bagged in a sub-dir of the temp dir.